src/Inc/Menus.php

```diff
@@ -0,0 +1,61 @@
+<?php namespace Leean\Endpoints\Inc;
+
+/**
+ * Class Menus
+ *
+ * @package Leean\Endpoints\Inc
+ */
+class Menus
```

Comment:
I was able to see the menu items on this just this.

http://snaps.getmoxied.net/0E1a3t2i0G0B

http://snaps.getmoxied.net/3B0J2C0c2T3n